### PR TITLE
New package: DisplayLink.GraphicsDriver version 10.2.6841.0

### PR DESCRIPTION
**Pull request by HandyHat**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56298)

### CODE REVIEW

File: manifests/s/Synaptics/DisplayLinkGraphicsDriver/10.2.6841.0/Synaptics.DisplayLinkGraphicsDriver.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Synaptics.DisplayLinkGraphicsDriver
+PackageVersion: 10.2.6841.0
+PackageLocale: en-US
+Publisher: Synaptics Incorporated
```
Comment (OfficialEsco) [inline manifests/s/Synaptics/DisplayLinkGraphicsDriver/10.2.6841.0/Synaptics.DisplayLinkGraphicsDriver.locale.en-US.yaml:7]:
```suggestion
Publisher: DisplayLink Corp.
```

File: manifests/s/Synaptics/DisplayLinkGraphicsDriver/10.2.6841.0/Synaptics.DisplayLinkGraphicsDriver.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+PublisherUrl: https://www.synaptics.com/
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+# Author: 
+PackageName: DisplayLink USB Graphics Software
```
Comment (OfficialEsco) [inline manifests/s/Synaptics/DisplayLinkGraphicsDriver/10.2.6841.0/Synaptics.DisplayLinkGraphicsDriver.locale.en-US.yaml:12]:
```suggestion
PackageName: DisplayLink Graphics
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
The DisplayName and Publisher needs to match whats in `A`dd or `R`emove `P`rograms

**Review by OfficialEsco — DISMISSED** [review]

**Comment by HandyHat** [comment]
Does the package identifier also need to be changed? @OfficialEsco

**Comment by OfficialEsco** [comment]
> Does the package identifier also need to be changed? @OfficialEsco

Its not a big deal, as a user of the software, does `Synaptics` or `DisplayLink` make the most sense to you? If you go for the change you can probably do `DisplayLink.GraphicsDriver`

**Comment by HandyHat** [comment]
> > Does the package identifier also need to be changed? @OfficialEsco
> 
> Its not a big deal, as a user of the software, does `Synaptics` or `DisplayLink` make the most sense to you? If you go for the change you can probably do `DisplayLink.GraphicsDriver`

Displaylink would make more sense in my opinion. Thanks!

**Review by OfficialEsco — DISMISSED** [review]

**Review by OfficialEsco — APPROVED** [review]
Thanks for doing that, however `Validation-Domain` is caused by the Publisher not matching the URL, which means the Microsoft Team will have to validate that the url is from where its supposed to (you can comment on how to find the url on the website after the validation, it will remove the Needs-Author-Feedback label)

**Comment by HandyHat** [comment]
The download link can be accessed at https://www.synaptics.com/products/displaylink-graphics/downloads/windows

**Pull request merged**